addons/editor-dark-mode/addon.json

```diff
@@ -1002,6 +1002,34 @@
         "darkScrollbars": true
       }
     },
+    {
+      "name": "Scratch 2.0 Style",
```

Comment:
Maybe a better name is `Scratch 2.0 colors`?

Reply:
Why not just "Scratch 2.0"?